src/db.ts

```diff
@@ -748,7 +749,6 @@ function validateDatabaseName(databaseName: string) {
   const invalidChars = [' ', '.', '$', '/', '\\'];
   for (let i = 0; i < invalidChars.length; i++) {
     if (databaseName.indexOf(invalidChars[i]) !== -1)
-      // TODO(NODE-3405): Change this out for a child of MongoParseError
```

Comment:
Tag NODE-3484 here

Reply:
NODE-3484 should be tagged at the top of this function since the whole thing has to be moved into MongoDBNamespace. Should I tag it here instead?

Reply:
If it's already tagged, then ignore this.